mock-core-configs/etc/mock/almaepel-8-aarch64.cfg

```diff
@@ -0,0 +1,6 @@
+include('templates/almalinux-8.tpl')
+include('templates/epel-8.tpl')
+
+config_opts['root'] = 'almaepel-8-aarch64'
```

Comment:
Can we go ahead an introduce these as `alma-epel-8-*`?

Reply:
That's going to mess with the parsing for all the stuff that auto-select configs (copr plugin, fedpkg, etc.)

Reply:
I've called it `alma+epel-8-*` instead, but that's now done.